logging/env_logger.h

```diff
@@ -57,12 +57,12 @@ class EnvLogger : public Logger {
       // Preserve iostats not to pollute writes from user writes. We might
       // need a better solution than this.
       SetPerfLevel(PerfLevel::kDisable);
-      iostats_context.disable_iostats = true;
+      IOSTATS_DISABLE(true);
```

Comment:
Nit: Shouldn't this be IOSTATS_SET_DISABLE, to conform to the signature/meaning of the other methods?